backend/src/main/java/ai/verta/modeldb/experimentRun/ExperimentRunServiceImpl.java

```diff
@@ -1226,6 +1228,12 @@ public void getUrlForArtifact(
       roleService.validateEntityUserWithUserInfo(
           ModelDBServiceResourceTypes.PROJECT, projectId, ModelDBServiceActions.READ);
 
+      if (App.getInstance().getTrialEnabled() && request.getPartNumber() != 0) {
```

Comment:
only have the follwing restrictions
in trial do not support PUT
in trial in response to GetUrlForArtifact return false for multipartUploadOk